src/adv.py

```diff
@@ -49,3 +53,11 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
```

Comment:
You are off to a great start with your adv.py , please reach out to the help channel or myself and any other PM for assistance if you need help

I like you added the option for the player to enter their name